### DISCUSSION TITLE
Notice for project homepage temporarily unavailable

### DISCUSSION THREAD

**Discussion by tobychui** [tobychui/zoraxy#325] [Announcements]
## Notice for project homepage temporarily unavailable

![圖片](https://github.com/user-attachments/assets/ce6c65fd-40df-449c-b29d-78135470eb79)

As Gandi is trying to scam me for paying $300 hkd for renewing a domain that suppose to be around $130 hkd per year, I am currently planning to purchase a new domain for this project with a new domain name supplier.

Hence, within the following week(s), the zoraxy project homepage is not available for access. I am still looking for a good domain name supplier that can do easy auto-renew and low prices for .com domains. If anyone have any ideas, feel free to let me know :) 

Toby

### Updates
With the help from my professional IT friends, now the new project homepage has been migrated successfully and working like a charm. You can check out the new domain and homepage at [https://zoraxy.aroz.org](https://zoraxy.aroz.org) . Thanks so much for everyone involved!

**Comment by Tivin-i** [comment]
Cloudflare promises to have it priced at cost, but you have to use Cloudflare then...
Otherwise Namesilo has been good for me, not always the cheapest but they are consistent and hassle free.

**Comment by Morethanevil** [comment]
Glad you found a new domain ☺️

**Comment by Freekers** [comment]
You can compare TLD prices here, including renewal prices: https://tld-list.com/
